tests/rptest/archival/s3_client.py

```diff
@@ -75,10 +79,11 @@ def create_bucket(self, name):
                 raise err
 
     def delete_bucket(self, name):
+        self.logger.info(f"Deleting bucket {name}...")
         try:
             self._cli.delete_bucket(Bucket=name)
         except Exception:
-            self.logger.warn("Error deleting bucket, contents:")
+            self.logger.warn(f"Error deleting bucket {name}, contents:")
```

Comment:
It would be good to capture and print the exception here.

Reply:
Good point, added that in a separate line (they can be long) separately to the precursor to the contents list.